src/main/java/io/cloudsoft/winrm4j/winrm/WinRmTool.java

```diff
@@ -29,11 +29,21 @@ public WinRmToolResponse executeScript(List<String> commands) {
         return responseToWinRmToolResponse(response);
     }
 
+    public WinRmToolResponse executeCommand(String command) {
+        Response response = session.run_cmd(command);
+        return responseToWinRmToolResponse(response);
+    }
+
     public WinRmToolResponse executePs(List<String> commands) {
         Response response = session.run_ps(compileScript(commands));
         return responseToWinRmToolResponse(response);
     }
 
+    public WinRmToolResponse executePsCommand(String command) {
```

Comment:
We should include some javadoc to say what this command does (it's unfortunate that there wasn't any javadoc on the original `executePs(List)` method).